dashboard/src/components/partials/ChecksList/ChecksListItem.js

```diff
@@ -87,25 +90,12 @@ class CheckListItem extends React.Component {
               <React.Fragment>
                 <Code>{check.command}</Code>
                 <br />
-                Executed{" "}
-                <strong>
-                  {check.interval ? (
-                    `
-                      every
-                      ${check.interval}
-                      ${check.interval === 1 ? "second" : "seconds"}
-                    `
-                  ) : (
-                    <CronDescriptor expression={check.cron} />
-                  )}
-                </strong>{" "}
-                by{" "}
-                <strong>
-                  {check.subscriptions.length}{" "}
-                  {check.subscriptions.length === 1
-                    ? "subscription"
-                    : "subscriptions"}
-                </strong>.
+                {check.publish && <CheckPublishInfo check={check} />}
```

Comment:
🤔 how would you feel about naming this component CheckSummary or CheckDescriptor and including the unpublished case as well?

Reply:
Seems like a good idea, I'll add that.